### PR TITLE
DOC-2242: Added 6.8.3-specific entry to `changelog.adoc`.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2242

Changes:
* Added 6.8.3-specific entry to `changelog.adoc` for TinyMCE `6.8.3` community release.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `.api-version` bump before merging.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.8.3 - 2024-<month>-<date>
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:7]: - [ ] update date, before merging once confirmed.

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+
+=== Fixed
+* `ShadowDOM` skin was not loaded properly when used with js bundling feature.
+* The floating toolbar would not be fully visible when the editor was placed inside a scrollable container.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/changelog.adoc:11]:
```suggestion
* The floating toolbar would not be fully visible in fullscreen mode when the editor was placed inside a scrollable container.
```

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ This project is built using [Antora](https://antora.org/).
 If you would like to contribute to the TinyMCE project please read the TinyMCE Documentation Contributor’s Guide at either:
 
 - [TinyMCE Documentation - Contributor's Guide](https://www.tiny.cloud/docs/configure/contributing-docs/).
-- [GitHub - Contributing to the TinyMCE developer documentation](https://github.com/tinymce/tinymce-docs/blob/develop/CONTRIBUTING.md#contributing-to-the-tinymce-developer-documentation).
+- [GitHub - How to contribute to TinyMCE’s documentation](https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#how-to-contribute-to-tinymces-documentation).
```
Comment (kemister85) [inline README.md:16]: @FarzadHayat we should point this link to `release` not `staging` for public pull requests.

I've added the link, but check confirm on your side that the link and anchor works as expected.

```suggestion
- [GitHub - How to contribute to TinyMCE’s documentation](https://github.com/tinymce/tinymce-docs/blob/release/docs-6/CONTRIBUTING.md#how-to-contribute-to-tinymces-documentation).
```

File: modules/ROOT/pages/changelog.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:7]:
```suggestion
== 6.8.3 - 2024-02-08
```

File: modules/ROOT/pages/changelog.adoc [hunk 4]
```diff
@@ -4,6 +4,12 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.8.3 - 2024-02-08
+
+=== Fixed
+* `ShadowDOM` skin was not loaded properly when used with js bundling feature.
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/changelog.adoc:10]:
```suggestion
* `ShadowDOM` skin was not loaded properly when used with JS bundling feature.
```

**Review by tiny-ben-tran — APPROVED** [review]

**Review by FarzadHayat — APPROVED** [review]

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc [hunk 5]
```diff
@@ -4,6 +4,12 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.8.3 - 2024-02-08
+
+=== Fixed
+* `ShadowDOM` skin was not loaded properly when used with JS bundling feature.
+* The floating toolbar would not be fully visible when the editor was placed inside a scrollable container.
+
```
Comment (kemister85) [inline modules/ROOT/pages/changelog.adoc:12]:
```suggestion
== 6.8.3 - 2024-02-08

=== Changed
* Update outbound TinyMCE website links.

=== Fixed
* `ShadowDOM` skin was not loaded properly when used with JS bundling feature.
* The floating toolbar would not be fully visible when the editor was placed inside a scrollable container.

```

**Pull request merged**